generator/idl_parser/idls.ts

```diff
@@ -88,6 +88,25 @@ export class InterfaceImpl implements Interface {
   }
 }
 
+export class InterfaceName {
+  name: string;
+
+  constructor(name) {
+    this.name = name;
+  }
+}
+
+export class InterfaceNames {
+  names: Array<InterfaceName>;
```

Comment:
i think InterfaceName class was not needed.
because in the future if you have some plan which other infomations add, it is fine.
but if it just had only interface's name, i think it is better use String instead of InterfaceName class.